src/ImageSharp/Metadata/Profiles/Exif/ExifReader.cs

```diff
@@ -64,11 +66,12 @@ private void GetThumbnail(uint offset)
                 return;
             }
 
-            var values = new List<IExifValue>();
+            List<IExifValue> values = new();
             this.ReadValues(values, offset);
 
-            foreach (ExifValue value in values)
+            for (int i = 0; i < values.Count; i++)
```

Comment:
Also wondering why this was changed.

Reply:
Because the compiler was complaining that an explicit cast was required for each item. The automatic fix was Linqs cast which was obviously poor from a perf perspective so a for loop was a better fix